lxd/container_lxc.go

```diff
@@ -2777,10 +2821,10 @@ func (c *containerLXC) Start(stateful bool) error {
 		return err
 	}
 
-	// Start proxy devices
-	err = c.restartProxyDevices()
+	// Run any post start hooks.
+	err = c.runPostStartHooks(postStartHooks)
 	if err != nil {
-		// Attempt to stop the container
+		// Attempt to stop container.
 		c.Stop(false)
```

Comment:
@stgraber although this code was already here, in my testing today, I've found it doesn't actually work. The Stop() function returns an error saying "A start operation is already in progress".

Can you suggest a way to stop a container during the Start() function? Thanks

Reply:
@stgraber as discussed, add op.Done here and that works.